### PR TITLE
Feat/settings/easier descriptions

### PR DESCRIPTION
**Pull request by blaskovicz**
Various improvements to facilitate the ease of developers adding settings descriptions, along with the ease of maintainers configuring the system.

Note that for settings or settings groups without a description, the name of the i18n key will be shown (until it's populated).

In this change, I specifically added english ldap settings labels as well.

Example of the ldap settings page:
image

### CODE REVIEW

**Comment by blaskovicz** [comment]
cc @gmsecrieru since we were talking about this in [!1236](https://github.com/RocketChat/Rocket.Chat/pull/1236/).

**Comment by gmsecrieru** [comment]
Looks good, thanks!

**Comment by blaskovicz** [comment]
As a follow-up to this pull request (once it's merged), I can start looking at allowing markdown in the settings descriptions (so examples, links, emphasis, etc are more clear). Thoughts?

**Comment by graywolf336** [comment]
Yes!! :+1: This has been something I've been wondering how best to do or even do in the current context, glad to see this. :smile:

**Comment by blaskovicz** [comment]
I can start looking at that this weekend pending any change I need to do before this is merged - @gmsecrieru

File: packages/rocketchat-lib/settings/server/methods.coffee [hunk 1]
```diff
@@ -18,7 +18,12 @@ RocketChat.settings.add = (_id, value, options = {}) ->
 	updateSettings =
 		i18nLabel: options.i18nLabel or _id
 
-	updateSettings.i18nDescription = options.i18nDescription if options.i18nDescription?
+  # default description i18n key will be the setting name + "_Description"
```
Comment (marceloschmidt) [inline packages/rocketchat-lib/settings/server/methods.coffee]: Just fix the indentation here and below, please? We use tabs for indenting.

**Comment by blaskovicz** [comment]
@marceloschmidt I fixed the whitespace in all the files and rebased on top of master.

**Pull request merged**